content/cli/orch_cli/secrets.md

```diff
@@ -160,17 +164,23 @@ Update an existing secret.
 One of these flags:
 
 * `-s, --secret-string TEXT` - The string to use as the secret's value.
-* `-f, --secret-file TEXT` - The name of the secret file that contains the value to be set.
+* `-f, --secret-file TEXT` - The secret's file to use its content as value to be set.
+                             You cannot use this argument with arguments: [secret_string].
+
+#### Optional flags:
 
+* `--hidden-value / --not-hidden-value` - The secret value is only shown to the user that created the secret and to admins. Use of the secret is allowed according to user roles and the visibility of the secret.
```

Comment:
The secret's value is only presented to the user who created it, tenants managers and sys-admins. Use of the secret is allowed according to the users roles, and visibility of the secret.